docs/action/ami.mdx

```diff
@@ -34,6 +34,17 @@ icon: "image"
 
   </Step>
 
+  <Step title="Change the Windows password">
+    Inside the Windows desktop environment:
+
+    1. Open up Powershell
+    2. Run C:\testdriver\RotateLocalPasswords.ps1
+    3. Copy the password presented in the output to a safe place. This is the only chance to copy the password. 
+    4. Restart the instance and double check you can log in before continuing. 
+
+    Note that this will change the password for the testdriver and rdp_local accounts, but Administrator will remain as a password you can only retreive through the AWS Ec2 UI. 
```

Comment:
1. Is it `Ec2` or `EC2`?
2. Is the `AWS EC2 UI` linkable?  https://console.aws.amazon.com/ec2/home redirects for me.
3. Can you put code references in back-ticks (e.g. Run `C:\testdriver\...`, `rdp_local`)

Reply:
Cool Updated. 
The UI is only in the UI for a specific instance. but I linked the instructions